diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index e594bbe..36e6c5a 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -467,10 +467,10 @@ module.exports = {
     return require('deprecated-react-native-prop-types').PointPropType;
   },
   get ViewPropTypes(): $FlowFixMe {
-    console.warn(
-      'ViewPropTypes will be removed from React Native. Migrate to ' +
-        "ViewPropTypes exported from 'deprecated-react-native-prop-types'.",
-    );
+    // console.warn(
+    //   'ViewPropTypes will be removed from React Native. Migrate to ' +
+    //     "ViewPropTypes exported from 'deprecated-react-native-prop-types'.",
+    // );
     return require('deprecated-react-native-prop-types').ViewPropTypes;
   },
 };
